install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -109,6 +109,8 @@ spec:
 {{- if .Values.traceSampling }}
           - name: PILOT_TRACE_SAMPLING
             value: "{{ .Values.traceSampling }}"
+          - name: PILOT_ENABLE_PROTOCOL_SNIFFING
```

Comment:
I think this is in the wrong block, needs it's own if - not traceSampling.

Reply:
+1